src/index.html

```diff
@@ -1,13 +1,399 @@
+              <img
+                alt="blonde with earbuds"
+                class="section__gallery-img"
+                src="/images/photos/speaker-2.jpg"
+              >
+            </div>
+            <div class="container-image">
+              <img
+                alt="blonde with earbuds"
+                class="section__gallery-img"
+                src="/images/photos/speaker-1.jpg"
+              >
+            </div>
+          </div>
+          <h3 class="section__subtitle">
+            Speakers
+          </h3>
+
+          <div class="section__gallery
+                      section__gallery--reverse">
+            <div class="container-image">
+              <img
+                alt="blonde with earbuds"
+                class="section__gallery-img"
+                src="/images/photos/sunglasses-1.jpg"
+              >
+            </div>
+            <div class="container-image">
+              <img
+                alt="blonde with earbuds"
+                class="section__gallery-img"
+                src="/images/photos/sunglasses-2.jpg"
+              >
+            </div>
+          </div>
+          <h3 class="section__subtitle">
+            Audio sunglasses
+          </h3>
+
+          <a
+            href="index.html#"
+            class="button"
+          > View all
+          </a>
+        </section>
+
+        <section
+          class="section"
+          id="id-why-buy-us"
+        >
+          <div class="why-buy-from-us">
+            <h2 class="section__title why-buy-from-us__title">
+              Why buy direct
+              <br>
+              from Bose
+            </h2>
+
+            <div class="why-buy-from-us__description">
+              <ul class="why-buy-from-us__list">
+                <li class="why-buy-from-us__item">
+                  Free 2-day shipping and returns
+                </li>
+                <li class="why-buy-from-us__item">
+                  90-day risk-free trial
+                </li>
+                <li class="why-buy-from-us__item">
+                  World class customer service
+                </li>
+                <li class="why-buy-from-us__item">
+                  My Bose account management
+                </li>
+              </ul>
+              <p class="why-buy-from-us__paragraph">
+                A great product is more than what’s in the box. It’s a promise
+                of premium performance, world-class support, and everything you
+                expect from a trusted brand. It’s just one of many reasons why
+                you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </section>
+
+        <section
+          class="section"
+          id="id-contact-us"
+        >
+          <h2 class="section__title">
+            Contact us
+          </h2>
+
+          <div class="contact-us">
+            <form
+              class="form contact-us__form"
+              action="index.html#"
```

Comment:
sending `POST` request to this endpoint causes _**405 Not Allowed**_ error
what you can do is the following piece of code
<img width="657" alt="Screenshot 2022-10-27 at 17 06 43" src="https://user-images.githubusercontent.com/75496546/198307632-0f015ec9-c7cc-45fc-b12d-5200ea6bb866.png">

little bit more information about `preventDefault` [here](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)

Reply:
Hi, thank you for the detailed feedback! I've implemented the required changes and re-deployed the website to GP. Let me know if anything else needs to be done!